Lesson_11/Tusk_3.py

```diff
@@ -0,0 +1,19 @@
+class MyCtxManager:
+
+    def __enter__(self):
+        print('=' * 10)
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        if exc_type:
+            print(exc_val)
+        print('=' * 10)
```

Comment:
Add here `return True` to avoid exception raising

Reply:
Thank you. I was trying to fix this, but I didnt know how.